src/App.jsx

```diff
@@ -2,14 +2,14 @@ import { useState } from 'react';
 import './App.scss';
 
 export const App = () => {
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
 
   const addOne = () => {
-    // write code here
+    setCount(a => a + 1);
```

Comment:
Consider to use better naming.
Will be good prevCount
```suggestion
    setCount(prevCount => prevCount + 1);
```